docs/manual/performance/http-caching.de.md

```diff
@@ -377,8 +377,86 @@ darüber in Kenntnis setzt. Dies kann bequem via `Cache-Control: no-store` errei
 Denn ansonsten rennen wir wiederum in das Problem, dass zu viele Cache-Einträge generiert werden.
 {{% /notice %}}
 
+## Shared Cache-Wartung und Cache-Tagging
+
+An dieser Stelle müssen wir noch ein weiteres Konzept erläutern. Als Anwender von Contao wirst du das gar nicht bemerken,
+es ist aber ein wichtiger Teil des hervorragenden Caching-Frameworks von Contao und ist daher nicht zuletzt auch ein
+tolles Verkaufsargument.
+
+Ein entscheidender Unterschied zwischen Shared Cache und Private Cache ist, dass der Ort des Shared Caches bekannt ist und
+darauf zugegriffen werden kann. Wir können also den Shared Cache aktiv bewirtschaften und einzelne Cache-Einträge oder den
+gesamten Cache löschen. Das können wir beim privaten Cache nicht.
+
+Wenn du also Änderungen am Inhalt vornimmst, kannst du jederzeit dafür sorgen, dass der Shared Cache geleert wird und
+Besucher somit die neuste Version deiner Webseite sehen. Dazu dient die Option **»Shared Cache leeren«** beim 
+Menüpunkt **»Systemwartung«**.
+
+Contao wäre aber nicht Contao, wenn du das bei jeder Änderung selber machen müsstest.
+
+Contao verfügt über ein Framework, das es Entwicklern erlaubt, mit »Cache Tagging« zu arbeiten. Beim Generieren der Antwort wird diese dazu mit Tags ausgezeichnet, sodass diese vom Cache Proxy als Metadaten
+zum Cache-Eintrag gespeichert werden können. Auf Basis dieser Information, können dann Einträge mit gewissen Cache Tags
+invalidiert (so nennt man den Lösch-Vorgang beim Caching) werden.
+
+Jede Antwort die Contao generiert enthält also jede Menge solcher Cache Tags. Eine Antwort könnte also bspw. so aussehen:
+
+````http request
+HTTP/1.1 200 OK
+
+Content-Type: text/html; charset=utf-8
+Content-Length: 42
+Cache-Control: max-age=3600
+X-Cache-Tags: contao.db.tl_page.18, contao.db.tl_layout.16, contao.db.tl_content.42, contao.db.tl_content.10
+
+<html>
+...
+</html>
+````
+
+Die Anzahl der Tags kann dabei beliebig wachsen. In diesem Beispiel hat Contao die Antwort mit drei Cache Tags ausgezeichnet
+und wie du vielleicht bereits bemerkt hast, enthalten diese die Information, dass es sich bei dieser Antwort um die
+Seite mit ID `18` mit Seitenlayout ID `16` handelt und sich die Inhaltselemente mit ID `42` und `10` darauf befinden.
+
+Und hier kommt's: Bearbeitest du eins dieser Elemente im Contao Backend, invalidiert Contao automatisch alle Cache-Einträge
```

Comment:
ich bin mir nicht sicher, ob nach dem Doppelpunkt «Bearbeitest» wirklich grossschgeschrieben wird?

Reply:
In meiner Welt schon, weil nach dem Doppelpunkt ein vollständiger Satz mit Prädikat und Subjekt folgt :)